static/src/js/components/Map/ProjectionSwitcher.js

```diff
@@ -1,46 +1,61 @@
-        <button
-          aria-label="North Polar Stereographic"
-          className="projection-switcher__button projection-switcher__button--arctic"
-          data-test-id="projection-switcher__arctic"
-          onClick={() => onChangeProjection(projections.arctic)}
-          title="North Polar Stereographic"
-          type="button"
-        />
-        <button
-          aria-label="Geographic (Equirectangular)"
-          className="projection-switcher__button projection-switcher__button--geo"
-          data-test-id="projection-switcher__geo"
-          onClick={() => onChangeProjection(projections.geographic)}
-          title="Geographic (Equirectangular)"
-          type="button"
-        />
-        <button
-          aria-label="South Polar Stereographic"
-          className="projection-switcher__button projection-switcher__button--antarctic"
-          data-test-id="projection-switcher__antarctic"
-          onClick={() => onChangeProjection(projections.antarctic)}
-          title="South Polar Stereographic"
-          type="button"
-        />
-      </div>
-    </Control>
-  )
-}
+class ProjectionSwitcherControl extends L.Control {
+  initialize(props) {
+    const { onChangeProjection } = props
+
+    this.onChangeProjection = onChangeProjection
+  }
+
+  options = {
+    position: 'bottomright'
+  }
+
+  onAdd() {
+    this.container = L.DomUtil.create('div', '')
+    const switcher = L.DomUtil.create('div', 'projection-switcher leaflet-bar', this.container)
+
+    const arcticButton = L.DomUtil.create(
+      'button',
+      'projection-switcher__button projection-switcher__button--arctic',
+      switcher
+    )
+    arcticButton.ariaLabel = 'North Polar Stereographic'
+    arcticButton.title = 'North Polar Stereographic'
+    arcticButton.setAttribute('data-testid', 'projection-switcher__arctic')
+    arcticButton.onclick = () => this.onChangeProjection(projections.arctic)
 
-ProjectionSwitcher.propTypes = {
-  onChangeProjection: PropTypes.func.isRequired
+    const geographicButton = L.DomUtil.create(
+      'button',
+      'projection-switcher__button projection-switcher__button--geo',
+      switcher
+    )
+    geographicButton.ariaLabel = 'Geographic (Equirectangular)'
+    geographicButton.title = 'Geographic (Equirectangular)'
+    geographicButton.setAttribute('data-testid', 'projection-switcher__geo')
+    geographicButton.onclick = () => this.onChangeProjection(projections.geographic)
+
+    const antarcticButton = L.DomUtil.create(
+      'button',
+      'projection-switcher__button projection-switcher__button--antarctic',
+      switcher
+    )
+    antarcticButton.ariaLabel = 'South Polar Stereographic'
+    antarcticButton.title = 'South Polar Stereographic'
+    antarcticButton.setAttribute('data-testid', 'projection-switcher__antarctic')
+    antarcticButton.onclick = () => this.onChangeProjection(projections.antarctic)
+
+    return this.container
+  }
+
+  onRemove() {
```

Comment:
is this an intentional over-ride?  If yes, I think a comment here could be helpful.